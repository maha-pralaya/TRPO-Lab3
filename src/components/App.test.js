import React from 'react';
import Sent from './App';
import renderer from 'react-test-renderer';

describe('Тестирование рендринга', () => {
    test('renders correctly', () => {
    const renderedApp = renderer.create(<Sent />);
    expect(renderedApp.toJSON()).toMatchSnapshot();
    });
});