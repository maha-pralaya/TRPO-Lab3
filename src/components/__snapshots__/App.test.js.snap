// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Тестирование рендринга renders correctly 1`] = `
<div>
  <textarea
    cols="40"
    id="mainText"
    name="mainText"
    onChange={[Function]}
    placeholder="Введите свой текст..."
    rows="6"
  />
  <div>
    <button
      id="clear"
      onClick={[Function]}
      type="button"
    >
      Очистить
    </button>
  </div>
  <p
    id="shortest"
  >
    Самое короткое предложение:
    пока не определено
  </p>
  <p
    id="longest"
  >
    Самое длинное предложение:
    пока не определено
  </p>
</div>
`;
